management/http.go

```diff
@@ -4,6 +4,9 @@ import (
 	"bytes"
 	"encoding/xml"
 	"fmt"
+	"log"
```

Comment:
general feedback. let me know if you wanna use logrus. https://github.com/Sirupsen/logrus it's a pretty good logging library.